service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -252,4 +252,17 @@ public DatasetPreviewTable getDatasetPreview(
     var dataset = datasetDao.retrieve(datasetId);
     return generateDatasetTablePreview(user, dataset, tableName);
   }
+
+  public void exportDataset(
+      AuthenticatedUserRequest user, DatasetId datasetId, String workspaceId) {
+    var dataset = datasetDao.retrieve(datasetId);
+    ensureActionPermission(user, dataset, SamAction.READ_ANY_METADATA);
```

Comment:
I'm not totally sure about this one, but I implemented the others.

Reply:
I agree that we shouldn't do a permission check here. At best it's duplication of effort, where we're doing the same check as the service; at worst we might prevent an operation that the service would allow.

Reply:
I think the general rule we've been following is to not check permissions if the underlying storage system is going to do it for us - it duplicates the check -> https://broadworkbench.atlassian.net/browse/DC-315 is the ticket that describes the why/history of that decision.

Reply:
Okay, I've removed this.